new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiDomain.java

```diff
@@ -119,24 +119,24 @@
 
   /**
    * Install Operator.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
    */
   @BeforeAll
-  public static void initAll() {
-    // create standard, reusable retry/backoff policy
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusbale retry/backoff policy
```

Comment:
It's a small point, but you changed "reusable" from the correct spelling to an incorrect one.

Reply:
Thanks for catching that. Your eyes are sharp :-)
Fixed.